packages/material-ui/src/FormControlLabel/FormControlLabel.d.ts

```diff
@@ -46,6 +51,12 @@ export interface FormControlLabelProps
    * @default 'end'
    */
   labelPlacement?: 'end' | 'start' | 'top' | 'bottom';
+  /**
+   * These props will be forwarded to the label typography component
+   * (as long as disableTypography is not `true`).
+   * @default {}
+   */
+  labelTypographyProps?: TypographyProps;
```

Comment:
I would encourage we don't add this prop, as we are moving in the `componentsProps` direction to solve the same use case, as a systemic feature.

```suggestion
```

#21453

Reply:
thanks! i'll check it out

Reply:
i've added `componentProps` to FormHelperText as well as FormComtrolLabel in the style of the linked code